tests/gtests/graph/unit/backend/elyzor/test_utils.hpp

```diff
@@ -26,6 +27,14 @@ namespace utils = dnnl::graph::tests::unit::utils;
 
 using ltsr_vec = std::vector<impl::logical_tensor_t>;
 
+// ONEDNN_ENABLE_ELYZOR_GRAPH_COMPILER_LIB_TESTS=0 - skip tests that require 'libgraph_compiler.so' loading
+#define SKIP_WHEN_NO_LIB_TESTS() \
+    if (!dnnl::impl::getenv_int_user("ENABLE_ELYZOR_GRAPH_COMPILER_LIB_TESTS", \
+                /*default_value=*/1)) { \
+        GTEST_SKIP(); \
+        return; \
+    }
+
```

Comment:
[`getenv_int_user()`](https://github.com/kurapov-peter/oneDNN/blob/a9e6d220461f8ad4b2898313cf246a8ab61e7c7c/src/common/utils.cpp#L96-L109) looks for variables with `ONEDNN_` prefix, that's why there's a difference between what we pass as an argument and which variable should actually be set